static/src/js/schemas/umm/ummSSchema.js

```diff
@@ -797,7 +797,7 @@ const ummSSchema = {
     ServiceTypeEnum: {
       description: 'This element is used to identify the type of the service.',
       type: 'string',
-      enum: ['OPeNDAP', 'THREDDS', 'WEB SERVICES', 'ESI', 'ECHO ORDERS', 'WCS', 'WMS', 'WMTS', 'EGI - No Processing', 'SOFTWARE PACKAGE', 'TOOL', 'WEB PORTAL', 'International Web Portal', 'MODEL', 'Harmony', 'ArcGIS Image Service', 'Web Feature Service', 'Web Geoprocessing Service', 'NOT PROVIDED']
+      enum: ['OPeNDAP', 'THREDDS', 'WEB SERVICES', 'ESI', 'ECHO ORDERS', 'WCS', 'WMS', 'WMTS', 'EGI - No Processing', 'SOFTWARE PACKAGE', 'TOOL', 'WEB PORTAL', 'International Web Portal', 'MODEL', 'Harmony', 'ArcGIS Image Service', 'Web Feature Service', 'Web Geoprocessing Service', 'NOT PROVIDED', 'SWODLR']
```

Comment:
Are these values sorted by MMT?

Reply:
No they are not

Reply:
To that end, 'NOT PROVIDED' should probably be the last thing in the list.